internal/controller/numaflowcontrollerrollout_controller_test.go

```diff
@@ -19,44 +19,46 @@ package controller
 import (
 	"context"
 	"strings"
+	"time"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	apiv1 "github.com/numaproj/numaplane/pkg/apis/numaplane/v1alpha1"
 )
 
 var _ = Describe("NumaflowControllerRollout Controller", Ordered, func() {
 	const (
-		namespace    = "default"
-		resourceName = "numaflow-controller"
+		namespace = "default"
+		duration  = 20 * time.Second
+		interval  = 1 * time.Second
```

Comment:
Why do you need to use different values that the default values defined in the suit_test.go file?

Reply:
We can also change the default values and check that they work for all tests.

Reply:
I'll update it accordingly, as it was not working earlier maybe It should work now after fixing the test cases.